operator/v1/types_console.go

```diff
@@ -1,9 +1,8 @@
 package v1
 
 import (
+	configv1 "github.com/openshift/api/config/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	"github.com/openshift/api/config/v1"
```

Comment:
We have a clear policy that imports should be grouped:
1. golang imports
2. other imports
3. k8s imports
4. origin imports

each group is separated with an empty line. Please follow these rules.

Reply:
Oop, I should have caught that, thanks! Updated.

Reply:
if it's a clear policy, where is it clearly stated?

(I'm all for that grouping but i've never seen it as anything more than a preferred convention).